app/controllers/transition/NeedToPayTaxYesNoController.scala

```diff
@@ -69,15 +71,14 @@ class NeedToPayTaxYesNoController @Inject()(
           val identifierType = request.userAnswers.identifierType
           Future.successful(BadRequest(view(formWithErrors, identifier, identifierType)))
         },
-        value =>
+        needsToPayTax =>
           for {
-            updatedAnswers <- Future.fromTry(request.userAnswers.set(NeedToPayTaxYesNoPage, value))
+            hasAnswerChanged <- Future.fromTry(Success(!request.userAnswers.get(NeedToPayTaxYesNoPage).contains(needsToPayTax)))
+            updatedAnswers <- Future.fromTry(request.userAnswers.set(NeedToPayTaxYesNoPage, needsToPayTax))
             _ <- playbackRepository.set(updatedAnswers)
-            - <- trustConnector.removeTransforms(request.userAnswers.identifier)
-            _ <- makeRequestIfConditionMet(value)(trustConnector.setTaxableTrust(request.userAnswers.identifier, value))
-            _ <- makeRequestIfConditionMet(value)(trustConnector.setTaxableMigrationFlag(request.userAnswers.identifier, value))
+            - <- updateTransforms(hasAnswerChanged, needsToPayTax)
```

Comment:
Can't tell if github is screwing up the formatting but this looks like a `-` instead of a `_`

Reply:
Sorted now